CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ endif()
 
 if(WIN32)
   # this is needed, because Windows can be weird, man.
-  set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+  set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS OFF)
```

Comment:
I'll double check but I moved this explicitly to the Engine as a property so that not _everything under the sun_ exports its symbols. OpenAL and libsndfile were being a real headache about it :/

Reply:
This needs to be done to Interop as well otherwise Windows breaks

Reply:
I think the default value for this is `OFF` fwiw

Reply:
Should be resolved on Windows now, Interop and Engine explicitly export symbols but everything else should not